protected/humhub/components/ModuleManager.php

```diff
@@ -270,6 +278,56 @@ public function getModules($options = [])
         return $modules;
     }
 
+    public function filterModules(array $modules, $filters = []): array
```

Comment:
Some PHPDoc would be nice

Reply:
https://github.com/humhub/humhub/pull/5476/commits/f4163d88cac591e645cc04817f763a0bd42ca509#diff-46a3b644fa0848e9b9ff89b55e78b267182b69d39cea2f61870c54d37d0a78a5R281